charts/launch-agent/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: launch-agent
 icon: https://em-content.zobj.net/thumbs/240/apple/354/rocket_1f680.png
 description: A Helm chart for running the W&B Launch Agent in Kubernetes
 type: application
-version: 0.11.9
+version: 0.12.0
```

Comment:
For my edification when do we do a minor vs patch increment?

Reply:
@KyleGoyette I tried to follow this https://semantic-release.gitbook.io/semantic-release/#how-does-it-work 
This is a new feature, so that might go in the `minor` patch category.
But I'm happy to revert to a fix and make it a patch release.